src/app/components/ScrollablePromo/index.jsx

```diff
@@ -26,13 +44,32 @@ const PromoWrapper = styled.div`
   }
 `;
 
-const ScrollablePromo = ({ blocks, blockGroupIndex }) => {
-  const { dir } = useContext(ServiceContext);
+const LabelComponent = styled.strong`
+  display: block;
+  ${({ script }) => script && getDoublePica(script)};
+  ${({ service }) => getSansRegular(service)}
+  margin-bottom: ${GEL_SPACING_TRPL};
+  color: ${C_SHADOW};
+`;
 
-  const eventTrackingData = {
-    componentName: `edoj${blockGroupIndex}`,
-    format: 'CHD=edoj',
-  };
+const getEventTrackingData = (optimizely, blockGroupIndex) => ({
+  componentName: `edoj${blockGroupIndex}`,
+  format: 'CHD=edoj',
+  ...(optimizely && { optimizely }),
+});
```

Comment:
As we're re-using this component, does this mean any existing scrollable promos on the page that aren't EOJ, will now fire Optimizely events when they shouldn't? We might need an additional check like this:
```jsx
const getEventTrackingData = (hasOptimizely, blockGroupIndex) => ({
  componentName: `edoj${blockGroupIndex}`,
  format: 'CHD=edoj',
  ...(hasOptimizely && { optimizely }),
});
...
const ScrollablePromo = ({ blocks, blockGroupIndex, isRecommendationType }) => {
...
  hasOptimizely = isRecommendationType && hasVariationKey && optimizely;
  
  const eventTrackingData = getEventTrackingData(
      hasOptimizely,
      blockGroupIndex,
  );
...
};
```

Reply:
Looking at the docs `createInstance` always returns an optimizely object; so it never be null. You'd need to do a check to see if variation is the correct one otherwise it will add optimizely tracking everytime.

Reply:
I think I fixed this by using isRecommendationType in this line 71:
<img width="707" alt="Screenshot 2022-03-10 at 11 25 34" src="https://user-images.githubusercontent.com/90621252/157652483-bd965955-c5fc-434a-9405-dff571556542.png">

Reply:
happy to change the check for the variation instead of checking if it is a recommendation.

Reply:
ok, now I am checking that we are getting `variation_3` as `variationKey` for Optimizely tracking